website/docs/api/kb.md

```diff
@@ -222,55 +145,14 @@ Restore the state of the knowledge base from a given directory. Note that the
 > #### Example
 >
 > ```python
-> from spacy.kb import KnowledgeBase
 > from spacy.vocab import Vocab
 > vocab = Vocab().from_disk("/path/to/vocab")
-> kb = KnowledgeBase(vocab=vocab, entity_vector_length=64)
+> kb = FullyImplementedKB(vocab=vocab, entity_vector_length=64)
 > kb.from_disk("/path/to/kb")
 > ```
 
 | Name        | Description                                                                                     |
 | ----------- | ----------------------------------------------------------------------------------------------- |
 | `loc`       | A path to a directory. Paths may be either strings or `Path`-like objects. ~~Union[str, Path]~~ |
+| `exclude`   | List of components to exclude. ~~Iterable[str]~~                                                |
 | **RETURNS** | The modified `KnowledgeBase` object. ~~KnowledgeBase~~                                          |
-
-## Candidate {#candidate tag="class"}
```

Comment:
By removing some sections from this page, we might be creating broken links on the discussion forum or Stackoverflow. Can we try to prevent this? Maybe have a note here about the refactor and a referral to the in memory KB docs?

Reply:
Good point. We'd only keep the title (`##`) - and underneath the referral - or all headers (`###`)?

Reply:
Added a suggestion. We could also leave the `Candidate` section in `kb.md` - moving it to a separate page merely felt more consistent, as it's in its own module now.

Reply:
It's not strictly necessary. We don't always follow the structure of the codebase so closely in the documentation. I'll have a look at your recent edits!